src/MockClassifier.Api/Services/TokenService.cs

```diff
@@ -0,0 +1,45 @@
+﻿using MockClassifier.Api.Models;
+using MockClassifier.Api.Interfaces;
+using System.Text.RegularExpressions;
+
+
+namespace MockClassifier.Api.Services
+{
+    public class TokenService : IClassifierService
+    {
+       private readonly Regex ministryRegex = new("<random>|<justice>|<education>|<defence>|<environment>|<cultural>|<rural>|<economic>|<finance>|<social>|<interior>|<foreign>", RegexOptions.Compiled);
```

Comment:
I'm not sure but is there any way this Regex can be derived from the `MinistryEnum` to avoid having them all written out again?